resource/pkg_factories.cpp.em

```diff
@@ -17,277 +17,43 @@
 @#    Mapping between messages as well as their fields
 @###############################################
 @
```

Comment:
Nit:

```
#include <memory>
#include <string>
```

Reply:
That isn't necessary of the included corresponding header includes them.